web/handlers/admin/members_test.go

```diff
@@ -112,12 +112,12 @@ func TestMembersAdd(t *testing.T) {
 	}
 	checkInputsAreDisabled(false)
 
-	// verify that inputs are disabled for RoleMember
+	// verify that inputs are enabled for RoleMember
 	ts.User = roomdb.Member{
 		ID:   7331,
 		Role: roomdb.RoleMember,
 	}
-	checkInputsAreDisabled(true)
+	checkInputsAreDisabled(false)
```

Comment:
That's a pretty counterintuitive way to check that inputs are enabled.

Reply:
Can see why you might say that but then it is done several times in the rest of the test suite so I just do the same :woman_shrugging: